js/modules/k6/http/request.go

```diff
@@ -127,7 +118,11 @@ func (h *HTTP) Request(ctx context.Context, method string, url goja.Value, args
 		return nil, err
 	}
 
-	return h.request(ctx, req)
+	resp, err := netext.Do(ctx, req)
```

Comment:
https://golang.org/pkg/net/http/#Client.Do

Reply:
Yes, though let's compare the full names - `net/http/Client.Do()` vs `lib/netext/Do()`. What are we `Do`-ing exactly? :wink:
And while I'm also a bit annoyed by the name in the stdlib (since you *make* a request, you don't *do* a request...), that's not the biggest issue here...